test/parallel/test-http-information-headers.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const http = require('http');
+const Countdown = require('../common/countdown');
+
+const test_res_body = 'other stuff!\n';
```

Comment:
```suggestion
const responseBody = 'other stuff!\n';
```

Just a nit only because we prefer camelCase on js but it can be ignored.

Reply:
This is almost an identical copy of another file, the same variable probably there too.